package.json

```diff
@@ -1,37 +1,34 @@
 {
   "name": "@aragon/os",
   "version": "5.0.0-beta.3",
+  "author": "Aragon Association <legal@aragon.org>",
+  "license": "(GPL-3.0-or-later OR MIT)",
+  "repository": "https://github.com/aragon/aragonOS",
   "description": "Core contracts for Aragon",
+  "contributors": [
+    "Jorge Izquierdo <jorge@aragon.one>"
+  ],
+  "files": [
+    "/abi",
+    "/build",
+    "/contracts"
+  ],
   "scripts": {
-    "compile": "truffle compile",
```

Comment:
I think the `compile` script is still useful, but curious if you think it isn't!